### ISSUE TITLE
Workflow

### ISSUE THREAD

**Issue by fenriques** [mworion/MountWizzard3-DIST#15]
hi again Michel,
i still get awful results with my GM1000HPS. Now the setup is very solid, all cables are tighten and the scope is solid. I get DEC errors even above 200arcses during modeling.
My thinking is that I miss something in the workflow. Here's what I do, please check if I'm doing something wrong:
- My observatory is remote controlled, so I cannot balance (about 0.3% on both axis) or correct polar alignment (11 arces in RA, 0.1 in DEC) or cancel the previous alignment from my handset.
- I start  running a 3 star polar alignment in 'initial model' tab. Now error is 11arcsec and 0.1arcsec.
- Then sync the mount using 'plate solve and sync mount' 
- Then run a full model with 48 points. After 20 min I get a map like the one attached. But if i run it again results are even worse.
![screenshot from 2018-11-18 10-33-03](https://user-images.githubusercontent.com/38842860/48676220-16686f00-eb64-11e8-94a4-1e50724640df.png)

Do I need to save, load or do anything else?
thanks
ferrante

**Comment by mworion** [comment]
Ferrante,

the result is definitely not good. If you have a model, which is reasonable, you could omit the initial model and work through full model directly. But still it's strange. There must be something wrong. Could you post the log ?

Michel

**Comment by mworion** [comment]
Hi Ferrante,

something really bad happened. There is some mismatch between the python and the underlying C implementation of PyQt. Never seen this before. Did you experience this with earlier versions as well ?
Did you try to start MW3 RC5 a second time ?

Will take a little bit.

Michel

**Comment by fenriques** [comment]
hi michel,
i was not aware of any mismatch. so I don't know if this happened with previous versions.
I restarted MW3 many times and the results for modeling are more or less the same.
Attached yesterday s log. COuld have the same issue.
ferrante

[mount.2018-11-17.log](https://github.com/mworion/MountWizzard3-DIST/files/2593213/mount.2018-11-17.log)

**Comment by fenriques** [comment]
btw here are the dependencies when I try to update MW3:
ferrante@astropi:~$ sudo python3.6 -m pip install mountwizzard3 --upgrade --no-cache-dir 
[sudo] password for ferrante: 
Requirement already up-to-date: mountwizzard3 in /usr/local/lib/python3.6/dist-packages (3.0rc5)
Requirement not upgraded as not directly required: PyQt5==5.11.2 in /usr/local/lib/python3.6/dist-packages (from mountwizzard3) (5.11.2)
Requirement not upgraded as not directly required: matplotlib==2.2.2 in /usr/local/lib/python3.6/dist-packages (from mountwizzard3) (2.2.2)
Requirement not upgraded as not directly required: wakeonlan>=1.0.0 in ./.local/lib/python3.6/site-packages (from mountwizzard3) (1.1.6)
Requirement not upgraded as not directly required: requests==2.18.4 in ./.local/lib/python3.6/site-packages (from mountwizzard3) (2.18.4)
Requirement not upgraded as not directly required: astropy==3.0.3 in ./.local/lib/python3.6/site-packages (from mountwizzard3) (3.0.3)
Requirement not upgraded as not directly required: numpy==1.15.2 in /usr/local/lib/python3.6/dist-packages (from mountwizzard3) (1.15.2)
Requirement not upgraded as not directly required: requests_toolbelt==0.8.0 in ./.local/lib/python3.6/site-packages (from mountwizzard3) (0.8.0)
Requirement not upgraded as not directly required: PyQt5_sip<4.20,>=4.19.11 in /usr/local/lib/python3.6/dist-packages (from PyQt5==5.11.2->mountwizzard3) (4.19.12)
Requirement not upgraded as not directly required: six>=1.10 in ./.local/lib/python3.6/site-packages (from matplotlib==2.2.2->mountwizzard3) (1.11.0)
Requirement not upgraded as not directly required: cycler>=0.10 in ./.local/lib/python3.6/site-packages (from matplotlib==2.2.2->mountwizzard3) (0.10.0)
Requirement not upgraded as not directly required: pytz in ./.local/lib/python3.6/site-packages (from matplotlib==2.2.2->mountwizzard3) (2018.4)
Requirement not upgraded as not directly required: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in ./.local/lib/python3.6/site-packages (from matplotlib==2.2.2->mountwizzard3) (2.2.0)
Requirement not upgraded as not directly required: python-dateutil>=2.1 in ./.local/lib/python3.6/site-packages (from matplotlib==2.2.2->mountwizzard3) (2.7.3)
Requirement not upgraded as not directly required: kiwisolver>=1.0.1 in /usr/local/lib/python3.6/dist-packages (from matplotlib==2.2.2->mountwizzard3) (1.0.1)
Requirement not upgraded as not directly required: certifi>=2017.4.17 in ./.local/lib/python3.6/site-packages (from requests==2.18.4->mountwizzard3) (2018.4.16)
Requirement not upgraded as not directly required: chardet<3.1.0,>=3.0.2 in ./.local/lib/python3.6/site-packages (from requests==2.18.4->mountwizzard3) (3.0.4)
Requirement not upgraded as not directly required: idna<2.7,>=2.5 in ./.local/lib/python3.6/site-packages (from requests==2.18.4->mountwizzard3) (2.6)
Requirement not upgraded as not directly required: urllib3<1.23,>=1.21.1 in ./.local/lib/python3.6/site-packages (from requests==2.18.4->mountwizzard3) (1.22)
Requirement not upgraded as not directly required: setuptools in /usr/lib/python3/dist-packages (from kiwisolver>=1.0.1->matplotlib==2.2.2->mountwizzard3) (20.7.0)
You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

**Comment by mworion** [comment]
Ferrante,

lets try to split the topics:

1) with RC5 there were some updates to packages (your last mail), so updates should work and should be installed. I Habe to check, what happened to the pypi package online. I uploaded RC6 (on pypi only).

2) The 18/11 log is full of errors I mentioned. It's always the same, normally they don't hurt. There is no really clue out in the programming world I found so far how to debug these issues. In the past I hardly could see them, I made some trick at least to be able to log them. This should go away with new start. There will be a new architecture in future which hopefully avoids this type of issues (long term)

3) The 17/11 log. has non of these errors (good), but what surprises me, that a reply of the mount computer sends data in a wrong order. I've seen that in my environment, too (months ago) made a log and send that to 10micron for checking. They put it on the issue list. So far no updates in firmware. I this case I had to reboot the mount and start over.

In result this might happen as I use mount computer data exchange on a heavy load bases (for being quick).

Hope this make things clearer. I wonder If you have set the log level to info or debug to geht more output. 

Michel

**Comment by fenriques** [comment]
hi Michel,
I upgraded to RC6 (see attachment). 
Debug level was 'warning', now set to 'info'.
I will give a quick try now and send you the new log

Thanks
ferrante
[upgradeRC6.txt](https://github.com/mworion/MountWizzard3-DIST/files/2601839/upgradeRC6.txt)

**Comment by mworion** [comment]
Hi Ferrante,

as I see, the modeling worked for initial and build data until you stopped it after 16 stars. The off values were from my point ok, there was no extraordinary deviations after solving. Still solving took sometime a little bit. I wonder which model would have been processed finally.

One remark: please do not plate solve and sync mount between initial and full model. This command is only for sync purpose if you don't want to make a model and want to sync just for this position.

With the quality of the data I've seen in the initial model, you should be able to start directly with full model.

Michel

**Comment by fenriques** [comment]
Hi Michel, 
Model would have been more or less the same as in the image shared in the first post of this thread. Rms around 50, with single deviations around 70 or 80 arcsec.
I had to stop it because snow was coming in.
Ok I will not sync while building model.
I will try fixing cables and other devices more tightly but I fear that's  not the issue. Then I will ask 10micron for help.
Thanks 
Closing this as it's not a MW but.
